MacroExamplesLib/Macros.swift

```diff
@@ -47,3 +47,44 @@ public macro wrapStoredProperties(_ attributeName: String) = #externalMacro(modu
 @attached(member, names: named(_storage))
 @attached(memberAttribute)
 public macro DictionaryStorage() = #externalMacro(module: "MacroExamplesPlugin", type: "DictionaryStorageMacro")
+
+public protocol Observable {}
```

Comment:
We should probably start splitting up this library into multiple files, but this looks great